backend/api/invoices/fetch.py

```diff
@@ -18,6 +18,9 @@
 from backend.models import Invoice, InvoiceItem
 
 
+default_sort_by = {"date_due": True, "id": True, "payment_status": True}
```

Comment:
We have multiple users in the app. If you store sorting direction in the back-end code then it will affect all users. I'd suggest to store sorting direction on front-end and back-end will only filter and send the data base on the given sorting direction.